subsys/mgmt/mcumgr/mgmt/src/mgmt.c

```diff
@@ -72,24 +72,35 @@ mgmt_find_handler(uint16_t group_id, uint16_t command_id)
 const struct mgmt_group *
 mgmt_find_group(uint16_t group_id)
 {
-	struct mgmt_group *group = NULL;
 	sys_snode_t *snp, *sns;
 
 	/*
 	 * Find the group with the specified group id
-	 * from the registered group list, if one exists
-	 * return the matching mgmt group pointer, otherwise return NULL
 	 */
 	SYS_SLIST_FOR_EACH_NODE_SAFE(&mgmt_group_list, snp, sns) {
 		struct mgmt_group *loop_group =
 			CONTAINER_OF(snp, struct mgmt_group, node);
 		if (loop_group->mg_group_id == group_id) {
-			group = loop_group;
-			break;
+			return loop_group;
 		}
 	}
 
-	return group;
+	return NULL;
+}
+
+const struct mgmt_handler *
+mgmt_get_handler(const struct mgmt_group *group, uint16_t command_id)
+{
+	if (command_id >= group->mg_handlers_count) {
+		return NULL;
+	}
+
+	if (!group->mg_handlers[command_id].mh_read &&
+		!group->mg_handlers[command_id].mh_write) {
```

Comment:
Fix alignment so it matches previous line (tabs at start then spaces to match alignment)

Reply:
Fixed